.github/workflows/warn-on-editing-requirements.yml

```diff
@@ -0,0 +1,43 @@
+name: Maintainer approval for requirement edits
+
+on:
+  pull_request_review:
+    types: [submitted]
```

Comment:
I thought the `pull_request` event has a type like approved but it doesn't seem to have it